modules/submitForm.js

```diff
@@ -0,0 +1,17 @@
+import BookList from './classes.js';
+
+const bookList = new BookList();
+
+export default function submitForm(event) {
```

Comment:
- [ ] Kindly consider using arrow function syntax from ES6 as it is a requirement for this milestone.